spacy_experimental/span_finder/tests/test_span_finder.py

```diff
@@ -68,6 +70,40 @@ def test_span_finder_component_span_lengths():
     )
 
 
+def test_span_finder_component_set_annotations_span_lengths():
+
+    test_min_length = 2
+    test_max_length = 3
```

Comment:
Can you parameterize this with checks for each as 0, each set, both set and also include an explicit check for the correct number of spans in each case (also checking the min/max for good measure).